.github/actions/setup-deploy/action.yaml

```diff
@@ -76,6 +76,13 @@ runs:
         # behalf.
         version: v3.8.2
 
+    # Pin kubectl version to 1.23 otherwise interactions with k8s clusters versioned <=1.21 won't work.
+    # See https://github.com/2i2c-org/infrastructure/issues/1271.
+    - uses: azure/setup-kubectl@v2.0
+      with:
+        version: "1.23"
+      id: install
```

Comment:
```suggestion
```

The workflow has a jobs, the jobs has steps, this step has an id set to install. This is only relevant if we want to reference the outputs of this step or similar later, which we currently don't. Due to that, let's not add an id for this step.

Reply:
Oh, right! Copy-pasta gone wrong 😅 Thanks @consideRatio